Manifests/ManifestsApple/com.apple.syspolicy.kernel-extension-policy.plist

```diff
@@ -126,6 +126,20 @@
 			<key>pfm_type</key>
 			<string>boolean</string>
 		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<false/>
+			<key>pfm_description</key>
+			<string>Non-admin users can approve kernel extensions not explicitly allowed by configuration profiles.</string>
+			<key>pfm_macos_min</key>
+			<string>11</string>
+			<key>pfm_name</key>
+			<string>AllowNonAdminUserApprovals</string>
+			<key>pfm_title</key>
+			<string>Allow non-admin users to approve kernel extensions</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
+		</dict>
```

Comment:
This was something I thought about a while ago, but ultimately did not work as I hoped.

My only hesitation with adding this key in the current state of ProfileCreator is within a dictionary there is no way to restrict a preference given a specified/desired macOS version.  This means that anyone running less than Big Sur will get this key added to their profile.  I haven't tested enough to know whether including this on a non-Big Sur Mac is problematic.

I guess there's no real way around this, but wanted to put it out there.

Reply:
I don't think that this will happen, the key is directly under the root subkeys array. Did I miss anything?